docs/standard/conventions.md

```diff
@@ -0,0 +1,87 @@
+# Schema conventions
+
+This page documents conventions that MUST or SHOULD be applied when authoring OCDS schema or extensions.
+
+## Definitions section
+
+The top level of the schema is split between `properties` (the basic structure of an OCDS release) and `definitions` (containing objects that may be re-used in multiple locations across the schema) which are referenced whenever they are used. 
```

Comment:
"the basic structure of an OCDS release": Not true of the package schemas. Maybe just omit the parenthetical.